daffodil-test/src/test/resources/org/apache/daffodil/section13/boolean/boolean.tdml

```diff
@@ -0,0 +1,192 @@
+                </xs:sequence>
+            </xs:complexType>
+          </xs:element>
+        </xs:sequence>
+      </xs:complexType>
+    </xs:element>
+
+  </tdml:defineSchema>
+
+  <!--
+  Test name: booleanDefault
+  Schema: BooleanDefaultSchema, root booleanDefault
+  Purpose: This test demonstrates parsing of valid boolean default values
+-->
+
+  <tdml:parserTestCase name="booleanDefault" root="root1"
+                       model="BooleanDefaultSchema"
+                       roundTrip="onePass"
+                       description="Test valid boolean default values.">
+    <tdml:document><![CDATA[default1,default2,default3,default4
+false,true,0,1
+]]></tdml:document>
+    <tdml:infoset>
+      <tdml:dfdlInfoset>
+        <ex:root1>
+          <record>
+            <default1>false</default1>
+            <default2>true</default2>
+            <default3>false</default3>
+            <default4>true</default4>
+          </record>
+        </ex:root1>
+      </tdml:dfdlInfoset>
+    </tdml:infoset>
+  </tdml:parserTestCase>
+
+  <tdml:defineSchema name="BooleanDefaultSDE" elementFormDefault="unqualified">
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd" />
+    <dfdl:format ref="tns:GeneralFormat" separator=","
+                 initiator="" terminator="" leadingSkip='0' textTrimKind="none" initiatedContent="no"
+                 alignment="implicit" alignmentUnits="bits" trailingSkip="0" ignoreCase="no"
+                 separatorPosition="infix" occursCountKind="implicit" emptyValueDelimiterPolicy="both"
+                 representation="text" textNumberRep="standard" lengthKind="delimited" encoding="ASCII" />
+
+    <xs:element name="root1" dfdl:initiator="default1%NL;%WSP*;">
+      <xs:complexType>
+        <xs:sequence dfdl:separator="%NL;" dfdl:separatorPosition="postfix">
+          <xs:element name="record" maxOccurs="unbounded">
+            <xs:complexType>
+              <xs:sequence dfdl:separator=','>
+                <xs:element name="default1" type="xs:boolean" default="FALSE" dfdl:textBooleanTrueRep="TRUE" dfdl:textBooleanFalseRep="FALSE" />
+              </xs:sequence>
+            </xs:complexType>
+          </xs:element>
+        </xs:sequence>
+      </xs:complexType>
+    </xs:element>
+
+  </tdml:defineSchema>
+
+  <!--
+  Test name: booleanDefaultSDE
+  Schema: BooleanDefaultSchemaSDE, root booleanDefault
+  Purpose: This test demonstrates parsing of invalid default values
+-->
+
+  <tdml:parserTestCase  name="booleanDefaultSDE" root="root1"
+                        model="BooleanDefaultSDE"
+                        description="Test SDE due to SAXParseException resulting from invalid default value">
+    <tdml:document><![CDATA[default1
+TRUE
+]]></tdml:document>
+    <tdml:errors>
+      <tdml:error>Schema Definition Error</tdml:error>
+      <tdml:error>Invalid value constraint value</tdml:error>
```

Comment:
Is this test verifying that "FALSE" is not a legal default value for a boolean? Does the error message contain "FALSE" and the word "default" ? I think you should check for those here if possible.